frame/scored-pool/src/lib.rs

```diff
@@ -409,7 +422,7 @@ pub mod pallet {
 		#[pallet::weight(0)]
 		pub fn change_member_count(origin: OriginFor<T>, count: u32) -> DispatchResult {
 			ensure_root(origin)?;
-			MemberCount::<T, I>::put(&count);
+			Self::update_member_count(count)?;
 			Ok(())
```

Comment:
```suggestion
			Self::update_member_count(count)
```

There is one unused `std::cell::RefCell` import in the mock file which makes the CI unhappy.